### ISSUE TITLE
ulnar translocation of the carpus

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10731]
**Preferred term label:**
ulnar translocation of the carpus

**Synonyms**

**Definition (free text, please give PubMed ID)**
Anteroposterior wrist radiograph showing ulnar translocation of the carpus from the distal radius with widening of the radial styloid - scaphoid space

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Carpal instability nondissociative (CIND)

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001202
- label: Ulnar translocation of the carpus
- parents: [Abnormal carpal morphology (HP:0001191)](https://hpo.jax.org/app/browse/term/HP:0001191)
- pmid: PMID:35415594
- def: Anteroposterior wrist radiograph showing ulnar translocation of the carpus from the distal radius with widening of the radial styloid-scaphoid space.
- comment: This finding may be observed with Carpal instability nondissociative (CIND).
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10731
- synonyms:
